translator/tocwconfig/tocwconfig_test.go

```diff
@@ -101,11 +108,32 @@ func TestWindowsEventOnlyConfig(t *testing.T) {
 }
 
 func TestStatsDConfig(t *testing.T) {
-	resetContext(t)
-	expectedEnvVars := map[string]string{}
-	checkTranslation(t, "statsd_config", "linux", expectedEnvVars, "_linux")
-	checkTranslation(t, "statsd_config", "windows", expectedEnvVars, "_windows")
-	checkTranslation(t, "statsd_config", "darwin", nil, "_linux")
+	testCases := map[string]testCase{
```

Comment:
Made this into a for loop test so I could reset the context before each run of this test